obs-server.sls

```diff
@@ -22,11 +39,141 @@ install apache:
       - libapache2-mod-xforward
       - memcached
 
-create self signed ssl for testing:
+create_self_signed_ssl_for_testing:
   cmd.script:
     - name: generate_ssl.sh
     - source: salt://llvm-obs/obs-scripts/generate_ssl.sh
 
-rake task setup:
+install_obs_build_from_backports:
+  pkg.latest:
+    - pkgs:
+      - obs-build
+    - fromrepo: stretch-backports
+
+install_libsolv_from_testing:
```

Comment:
Please add a comment (or link to your blog post) explaining why we are doing that.

Reply:
Done